### DISCUSSION TITLE
Task List Feedback: Not all checkboxes are task lists

### DISCUSSION THREAD

**Discussion by DanielJette** [community/community#5197] [Projects and Issues]
We use the checkbox markdown element to denote a sort of pre-flight checklist on our PR templates. As in, we ask the developers to check off the relevant items when building their PR description.

In those cases, it does not make sense to track progress or "tasks done" as these elements are not tasks. It would be nice to be able to distinguish checklist items from task list items.

e.g.

- [ ] Requires QA
- [x] Requires documentation
- [ ] Requires release notes

**Comment by Zadjil-AR** [comment]
This suggestion also appears in #5238 and #5214
The one in #5238 includes a good suggestion of the syntax that could be used.

**Comment by potiuk** [comment]
Same here. This is really needed to be able to treat "checkboxes" that come from .yml issue templates might be "not treated" as tasks!

We are doing it for example to ask a question:

- [ ] Are you willing to submit a PR? (https://github.com/apache/airflow/pull/17855)

Hardly a completable task.

**Reply by RCheesley** [reply]
Same here, we use a 'do you accept the code of conduct' check box on our issue submission form, which is not a task - but the issue shows as having one task completed which can be confusing (example: https://github.com/mautic/mautic/issues/10371)

**Comment by omerbensaadon** [comment]
Seems this and https://github.com/community/community/discussions/5238 are dupes of eachother. 

We are working on this! Pinky promise

**Reply by RCheesley** [reply]
Great to hear that @omerbensaadon ! 🚀

**Reply by ross-spencer** [reply]
How is that pinky promise going? :wink:

**Reply by esafak** [reply]
It's not. The guy is out and will likely retire before Microsoft does anything.

**Comment by anthonyalayo** [comment]
Can we get an update? 2 years is a long time to wait for something like this. It would be pretty helpful for a lot of use cases.

**Reply by omerbensaadon** [reply]
https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists

We're going to release this feature in the next few months which should resolve this friction 👍🏼 HTML checkboxes will no longer count as tasks once tasklists are released

**Reply by tobias-feil-plana** [reply]
It looks like tasklists have been released. In our team, we are currently using a tasklist for what should be a checkbox. 
> HTML checkboxes will no longer count as tasks once tasklists are released

How do I create a HTML checkbox?

**Reply by ceztko** [reply]
In this [issue](https://github.com/podofo/podofo/issues/108) opened on October 31st the checkboxes were still treated as tasks. I am also interested in knowing when/how I will be able to use checkboxes without showing them as tasks.

**Reply by ross-spencer** [reply]
Naively creating HTML check boxes doesn't seem to render the check box portion at all. Any tips on the syntax needed here?

**Reply by jaredfrees-pg** [reply]
I am also interested in using checkboxes without showing them as tasks. HTML check boxes don't render in GitHub but they render on markdown preview. 
`- <input type="checkbox"/> 💥 Breaking change  (Breaks existing functionality)`

**Comment by ocornut** [comment]
> https://docs.github.com/en/issues/tracking-your-work-with-issues/about-tasklists
> 
> We're going to release this feature in the next few months which should resolve this friction 👍🏼 HTML checkboxes will no longer count as tasks once tasklists are released

@omerbensaadon AFAIK in spite of various replies it still doesn't seem possible to include checkboxes in an Issue Template that aren't treated as task lists when converted into an issue.  Using markdown `[ ]` checkboxes in issue template leads to uncheckable/read-only checkboxes, and the YAML "checkboxes" types leads to checkboxes turning into tasks.

TL;DR; it's not possible to use them in issue template to make users tick contents that doesn't need to be reflected in the the Issues or as Tasks.

**Comment by marcin-sg** [comment]
Since the tasklist blocks are [retired](https://github.com/orgs/community/discussions/5197), maybe this can be finally solved?
